crates/node-types/Cargo.toml

```diff
@@ -13,3 +13,8 @@ essential-hash = { workspace = true }
 essential-types = { workspace = true }
 serde = { workspace = true }
 serde_yaml = { workspace = true }
+tokio = { workspace = true, optional = true }
+
+[features]
+default = []
+new-block = [ "tokio" ]
```

Comment:
I wonder if maybe `block-notify` might be a better name for the feature?

Also, maybe we make it a default feature? Folks can always use `--no-default-features` if they don't want it, but I think the common case will be to require it?